### ISSUE TITLE
Beautiful component but can't get it to work

### ISSUE THREAD

**Issue by phutngo** [almogtavor/date-range-picker#27]
```
npm i dates-picker
```
Then can you show a simple functional component with import statement that we can use?
Even better show the code snippet from the demo page?

Thanks!

**Comment by phutngo** [comment]
@almogtavor any help/guidance would be much appreciated!

**Comment by almogtavor** [comment]
Yes there's an issue with the component and I'll publish the fix on version 1.1.0. I'll update here when it's done. If you want to use it now you can use version 1.0.18, but it won't work on React Strict Mode. So you'll be able to use it only if you remove strict mode. The usage will be as specified on the README.md (just import {DateRangePicker} from dates-picker)
